pkg/pulumiyaml/sort.go

```diff
@@ -37,16 +40,44 @@ func (e variableNode) key() *ast.StringExpr {
 	return e.Key
 }
 
-type configNode ast.ConfigMapEntry
+type configNode interface {
+	valueKind() string
+	key() *ast.StringExpr
```

Comment:
All this really means is we want `configNode` to implement `graphNode`. We can say that more directly:
```suggestion
	graphNode
```